docs/docusaurus/credentials/issue.md

```diff
@@ -22,24 +22,24 @@ Before using the Issuing Credentials protocol, the following conditions must be
 <Tabs groupId="vc-formats">
 <TabItem value="jwt" label="JWT">
 
-1. Issuer and Holder PRISM Agents up and running
-2. A connection must be established between the Issuer and Holder PRISM Agents (see [Connections](../connections/connection.md))
+1. Issuer and Holder Cloud Agents up and running
+2. A connection must be established between the Issuer and Holder Cloud Agents (see [Connections](../connections/connection.md))
 3. The Issuer must have a published PRISM DID, and the [DID document](/docs/concepts/glossary#did-document) must have at least one `assertionMethod` key for issuing credentials (see [Create DID](../dids/create.md) and [Publish DID](../dids/publish.md))
 4. The Holder must have a PRISM DID, and the DID document must have at least one `authentication` key for presenting the proof.
 
 </TabItem>
 <TabItem value="anoncreds" label="AnonCreds">
 
-1. Issuer and Holder PRISM Agents up and running
-2. A connection must be established between the Issuer and Holder PRISM Agents (see [Connections](../connections/connection.md))
+1. Issuer and Holder Cloud Agents up and running
+2. A connection must be established between the Issuer and Holder Cloud Agents (see [Connections](../connections/connection.md))
 3. The Issuer must have created an AnonCreds Credential Definition as described [here](../credentialdefinition/create.md).
 
 </TabItem>
 </Tabs>
 
 ## Overview
 
-The protocol described is a VC issuance process between two Atala PRISM Agents, the Issuer and the Holder.
+The protocol described is a VC issuance process between two Identus Cloud Agents, the Issuer and the Holder.
```

Comment:
This is not correct according to the replacement table, this shall be "Cloud Agent".